.github/workflows/rust-ci.yml

```diff
@@ -77,6 +77,7 @@ jobs:
         uses: actions-rs/cargo@v1
         with:
           command: test
+          args: --exclude iml-integration-tests --workspace
```

Comment:
Locally this might be inconvenient.

Reply:
Open to other suggestions.

Reply:
Maybe it shouldn't be implemented as a test, but a standalone executable?

Reply:
There is also `#[ignored]` and `cargo test -- --ignored`.

Reply:
My main concern is the possibility to step on the rake accidentally, in my view having these thing in a single repository and within single cargo workspace and use of the same testing framework make it more likely, but other than this I have no objections. I realized that I don't run tests ~~so I don't care~~ in a batch anyway (only per crate I am working on), thus it is ok :)